*** CMakeLists.txt	2018-12-05 17:15:36.888544599 +0100
--- CMakeLists.patched	2018-12-05 17:22:06.761859698 +0100
***************
*** 1,6 ****
--- 1,19 ----
  cmake_minimum_required(VERSION 3.5.1 FATAL_ERROR)
  project(CDFF VERSION 0.1 LANGUAGES C CXX)
  
+ find_program(CMAKE_CXX_CPPCHECK NAMES cppcheck)
+ if (CMAKE_CXX_CPPCHECK)
+   list(
+           APPEND CMAKE_CXX_CPPCHECK
+           "--enable=warning"
+           "--inconclusive"
+           "--force"
+           "--inline-suppr"
+           "--suppressions-list=${CMAKE_SOURCE_DIR}/Tools/CPPCheck/CppCheckSuppressions.txt"
+           "--output-file=CPPCheck-report.txt"
+   )
+ endif()
+ 
  # Maintainer: Romain Michalec <romain.michalec@strath.ac.uk>
  
  ## User interface --------------------------------------------------------------
