--- CMakeLists.txt.install_library	2018-11-15 23:23:10.479932688 +0000
+++ CMakeLists.txt.install_headers	2018-11-15 23:36:31.931917450 +0000
@@ -75,6 +75,7 @@
 		endforeach()
 	else()
 		install_shared( cloudcompare_core ${CMAKE_INSTALL_PREFIX}/lib 0 ) #default destination: /usr/local/lib
+		install(FILES ${header_list} DESTINATION "${CMAKE_INSTALL_PREFIX}/include/cloudcompare-core") #default destination: /usr/local/include/cloudcompare-core
 	endif()
 endif()
 
