--- CMakeLists.txt.rename_library	2018-11-15 23:20:33.151935679 +0000
+++ CMakeLists.txt.install_library	2018-11-15 23:23:10.479932688 +0000
@@ -74,7 +74,7 @@
 			install_shared( cloudcompare-core ${dest} 1 )
 		endforeach()
 	else()
-		install_shared( cloudcompare_core ${CMAKE_INSTALL_LIBDIR}/cloudcompare 0 ) #default destination: /usr/lib/cloudcompare
+		install_shared( cloudcompare_core ${CMAKE_INSTALL_PREFIX}/lib 0 ) #default destination: /usr/local/lib
 	endif()
 endif()
 
